lib/theme_settings_object_validator.rb

```diff
@@ -148,8 +185,13 @@ def is_property_present?(property_name)
   end
 
   def add_error(property_name, key, i18n_opts = {})
-    @errors[property_name] ||= ThemeSettingsObjectErrors.new
-    @errors[property_name].add_error(key, i18n_opts)
+    pointer = json_pointer(property_name)
+    @errors[pointer] ||= ThemeSettingsObjectErrors.new
+    @errors[pointer].add_error(key, i18n_opts)
+  end
+
+  def json_pointer(property_name)
+    "/#{@json_pointer_prefix}#{property_name}"
```

Comment:
I wonder if we should format the pointer in a way that is valid as a filter for the 'jq' command? I.e use dots instead of slashes?

Reply:
@OsamaSayegh does JSON Pointer support the dot notation? The slashes is following the json pointer format.

Reply:
Oh never mind then, I didn't know JSON Pointer was an industry standard (RFC6901).